dockerfiles/Dockerfile_linux_glibc_arm64_base

```diff
@@ -0,0 +1,9 @@
+FROM --platform=linux/arm64/v8 ubuntu:bionic-20220913
+
+RUN apt update
+
+RUN apt -y install make sudo
+
+RUN sudo apt -y install git wget build-essential rename
```

Comment:
Is it possible to pre-install this on the base image?

Reply:
These are the instructions for the base image.

Reply:
Ah, we just hadn't uploaded them until now?

Reply:
Yeah these were previously only in ECR as (built) images. I'm still going to build and upload these newer images; adding them to the repo is just a convenience so any of us can more easily update them again if that's ever necessary.

Reply:
Do we have a wiki page explaining how to update the images? Or better yet, should we write a make target that does it automatically?

Reply:
I just added a script that generates them with the correct tags.

I didn't add the login commands (it errors out if you aren't logged in) and it doesn't automatically push them (but it prints out the exact commands to run to push them), because I don't want people to replace the base images on accident. These should not be updated all that frequently, so it's not much of a roadblock.

Reply:
How are we supposed to know how to log in, then? I feel like copying and pasting instructions could get old after doing it two or three times, so how about a prompt, like "Upload to ECR? (y/n):"

Reply:
I was planning to add it to the release doc, but I just put in the login commands. Next commit will have the confirmation prompt.